### PR TITLE
cnao, Add components tier1 functests cont.

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds tier1 test lanes of the following components in CNAO: ovs-cni, macvtap

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml [hunk 1]
```diff
@@ -253,3 +253,67 @@ presubmits:
...
+      annotations:
+        fork-per-release: "true"
+      always_run: false
+      optional: true
+      decorate: true
+      skip_report: false
+      decoration_config:
+        timeout: 3h
+        grace_period: 25m
+      max_concurrency: 11
```
Comment (phoracek) [inline github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml:268]: See https://github.com/kubevirt/project-infra/pull/696#issuecomment-725340404

**Review by phoracek — CHANGES_REQUESTED** [review]

File: github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml (hunk 1 above)
Comment (RamLavi) [reply github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml:268]: good catch!

Comment (RamLavi) [reply github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml]: Done

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Comment by RamLavi** [comment]
/hold cancel

will continue debugging on the CNAO repo

**Pull request merged**